bHY6yh-d42838/cod-bqe7s7_bHY6yh-d42838.xml

```diff
@@ -0,0 +1,407 @@
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e671"> 
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e752">Ibidem xv epistola</ref> <quote xml:id="bHY6yh-d42838-Qd1e673">gloria regis est reperisse 
+               <lb ed="#P"/>iudices exquisitos / auari iudices nesciunt 
+               <lb ed="#P"/>quantum delinquunt / nam cum vendunt aliena 
+               <lb ed="#P"/>crimina sua faciunt esse pecchata.</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e683"> 
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e766">Ibidem libro viiii epistola viiii</ref> <quote xml:id="bHY6yh-d42838-Qd1e685">non mi<lb ed="#P" break="no"/>nus 
+               belli malum cum vitasse quam seua dis<lb ed="#P" break="no"/>cussionis 
+               euadere ipsa enim vera secu<lb ed="#P" break="no"/>ritas 
+               / de nulla iudicis iniquitate for<lb ed="#P" break="no"/>midat.</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e697">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e783">Ibidem xviiii epistola</ref> <quote xml:id="bHY6yh-d42838-Qd1e699">Metus cuncta disponit 
+               <lb ed="#P"/>sireus crimen in iudicante non inuenit</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e705">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e793">Ibidem libro x epistola vi</ref> <quote xml:id="bHY6yh-d42838-Qd1e707">Cum iustitia commit<lb ed="#P" break="no"/>titur
+               malis moribus non grauetur alioquin in<lb ed="#P" break="no"/>efficax 
+               est ab homine eligere quod agnoscitur 
+               <lb ed="#P"/>non habere.</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e718"> 
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e808">Ibidem xviiii</ref> <quote xml:id="bHY6yh-d42838-Qd1e720">illa sunt vera iudicia vbi 
+               <lb ed="#P"/>neminem comprimit ulla timiditas.</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e728"> 
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e820">Ibidem libro xi epistola ix</ref> <quote xml:id="bHY6yh-d42838-Qd1e730">Inutile est bon<lb ed="#P" break="no"/>um 
+               iudicem non accipere / et multis acci<lb ed="#P" break="no"/>piendi 
+               licentiam praebuisse.</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e738">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e832">Ibidem epistola xxxiii</ref> <quote xml:id="bHY6yh-d42838-Qd1e740">non est exitationis metus 
+               <lb ed="#P"/>vbi non est iudicis venalis auditus.</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e754"> 
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e850">Ibidem libro xii et fi: epistola primo</ref> <quote xml:id="bHY6yh-d42838-Qd1e756">Quam pericu
+               <cb ed="#P" n="b"/>
+               <lb ed="#P"/>losum est pati Iudicem Rationabiliter iratu
```

Comment:
"periculosum" is one word, so we need to keep it together.

```suggestion
               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e850">Ibidem libro xii et fi: epistola primo</ref> <quote xml:id="bHY6yh-d42838-Qd1e756">Quam pericu<cb ed="#P" n="b"/><lb ed="#P" break="no"/>losum 
               est pati Iudicem Rationabiliter iratu
```

Reply:
But it's over a column break. You want that on the same line too? I didn't think that would make sense.

Reply:
yes, let the word finish, and then add the hard return for readability.